creating/export-blender-to-aframe/index.md

```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
+http-server [full/path/to/index.html] -o
+
+There are other handy options for the server, specified on https://www.npmjs.com/package/http-server
+
+Make a directory to keep your blend file and exports
+We’ll come back to this directory later.
+
+
+# PART 2: GET AND CONVERT THE 3D MODEL
+---
+
+## Choose a blend file on SketchFab
+For this guide, I’m using this CV1 model: 
+https://sketchfab.com/models/591ff55a0f7b401cbb14fe454175b8b2
+Note that the model 
+
+## Download the blend file
+It will be a zip file.
+
+## Unzip/extract the blend file
+Right click on the .zip file and select Extract All. Save the extracted contents to a directory you can find easily.
+
+![Extract Zip Contents](https://github.com/WebVRRocks/webvrrocks/tree/master/creating/export-blender-to-aframe/img/ExtractAll.png)
+
+## Open the blend file with Blender
+Take a quick look at the Blender user interface. It has a high learning curve, but can be a powerful tool for creating 3D graphics. For now, we’ll just use it as an exporter for the CV1 model.
+
+## Export the model: File → Export → Wavefront (.obj)
+Name and save the .obj file (saved by default in same directory with the blend file)
+
+![Export .obj from Blender](https://github.com/WebVRRocks/webvrrocks/tree/master/creating/export-blender-to-aframe/img/ExportOBJ.png)
+
+## Copy the .obj and .mtl files into the directory with your index.html file
+Go back to the blend file in Blender and export as Collada (.dae)
+In the directory with your blend file, you should now have a .dae file, plus a .obj file and a .mtl file that goes with it. Note that there are a number of .jpg image files that came with the model. These are textures for the .obj: they are important! Copy-paste the .dae, .obj .mtl and all of the .jpg files into the directory with your index.html file.  
+
+
+# PART 3: WEB PAGE AND LOCAL SERVER
+---
+
+## Open your index.html file in your favorite text editor
+In the boilerplate, note that I’ve provided attribution to the model’s creator in a comment in lines 1-3. If you are new to A-Frame, there are some helpful comments in the code, as well.
+Because the 3D model files are in the same directory with your index.html, the file paths to them simply consist of their names. Go to line 26, which is within <a-assets>. This is  where we’ll initially load the assets for the scene. In src=”cv1_FF.obj” change cv1_FF.obj to match the name you gave your .obj file. Do the same for the .mtl and the .dae asset items in lines 27 and 28.
+Note that each model is given an ID, which is used to instantiate them into the scene as entities in lines 39 and 40. Using an asset system like this has the advantage of pre-loading your models, and you can then instantiate them as many times as you want without sacrificing as much performance.
+
+## Start up your local http server 
+Open the command prompt. Type:
+
+'''
+http-server [full\path\to\directory containing index.html] -o
```

Comment:
can you show your example path (e.g., C:\Users\myname\projects\export-blender-to-aframe\index.html`)?